python/tchannel/tornado/data.py

```diff
@@ -154,6 +159,17 @@ def __init__(
 
         self.scheme = scheme
 
+    @property
+    def status(self):
+        return self.flags
+
+    @status.setter
+    def status(self, status):
+        if status not in self.resp_status:
+            raise TChannelException("Not valid status!")
+
+        self.flags = status
```

Comment:
Once using an enum, map that to a flag manually. That will allow decoupling protocol level flags from response status codes.